services/github/github-package-lock-json.service.js

```diff
@@ -0,0 +1,126 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const {
+  isPackageLockJsonWithDependencies,
+  getLockDependencyVersion,
+} = require('../package-lock-json-helpers')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+const { BaseJsonService } = require('..')
+
+const keywords = ['npm', 'node']
+
+const dependencyQueryParamSchema = Joi.object({
+  filename: Joi.string(),
+}).required()
+
+class GithubPackageLockJsonDependencyVersion extends BaseJsonService {
```

Comment:
GitHub has a pretty strict throttling policy on non-authenticated requests, so all of the requests Shields issues to GitHub need to include authentication headers. We have additional base service class infrastructure to support this, which we'll want to use here (specifically `GithubAuthV3Service` to avoid the size limitations on the contents api)

```suggestion
class GithubPackageLockJsonDependencyVersion extends GithubAuthV3Service {
```